templates/admission_detail.html

```diff
@@ -205,103 +236,101 @@ <h4>
         let state_initial = "{{ admission.state }}";
         setStateActive(state_initial);
 
-        $(".dropdown-menu li a").click(function(){
-          let newState= $(this).attr("value");
-          if(newState === "Registration submitted") {
-          }else{
-              change_button_state(newState);
-          }
-          if(newState === "Waiting"){
-              let waiting_reason = {New: 'id_waiting_reason', Old: 'old_predefined_waiting_reason'};
-              let other_reason = {New: 'id_other_reason', Old: 'old_other_reason'};
-              openModal(
-                  'waiting_reason_modal',
-                  waiting_reason,
-                  other_reason
-              );
-          }
-          else if(newState === "Draft"){
-              openWarningModal(state_initial);
-          }
-          else if(newState === "Rejected"){
-              let refused_reason = {New: 'id_rejected-rejected_reason', Old: 'old_predefined_rejected_reason'};
-              let other_reason = {New: 'id_rejected-other_reason', Old: 'old_rejected_other_reason'};
-              openModal(
-                  'rejected_reason_modal',
-                  refused_reason,
-                  other_reason
-            );
-          }
-          else if(newState === "Registration submitted"){
-              check_mandatory_fields.call(this, newState);
-          }
-          else if(newState === "Accepted"){
-              let condition_acceptance = {New: 'id_condition_of_acceptance', Old: 'old_condition_of_acceptance'};
-              openModalAcceptance(
-                      'condition_acceptance_modal',
-                      condition_acceptance
-              );
-          }
-          else if(newState === "Cancelled"){
-              let cancelled_reason = {New: 'id_state_reason', Old: 'old_cancel_reason'};
-              openModal(
-                      'cancel_reason_modal',
-                      cancelled_reason
-              );
-          }
-          else{
-              $('#form').submit();
-          }
+        $(".dropdown-menu li a").click(function () {
+            console.log("YOLO");
+            document.getElementById('opened_tab').value = $('li.active')[0].id;
```

Comment:
autre suggestion: récupérer le tab actif à l'aide du this (tab sur lequel on a fait le click)